main.go

```diff
@@ -213,7 +109,7 @@ func main() {
 	log.Printf("- xcodebuild_version: %s (%s)", xcodebuildVersion.Version, xcodebuildVersion.BuildVersion)
 
 	// Detect xcpretty version
-	if configs.OutputTool == "xcpretty" {
+	if cfg.OutputTool == "xcpretty" {
```

Comment:
i think we should use our usual xcpretty handling here as well: https://github.com/bitrise-steplib/steps-xcode-build-for-simulator/pull/12/files#diff-7ddfb3e035b42cd70649cc33393fe32cR91

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-xcode-archive-mac/pull/29/commits/f1381763b2302ad0155519d0806f353c74e5761b